src/Models/HydrostaticFreeSurfaceModels/pcg_implicit_free_surface_solver.jl

```diff
@@ -44,8 +44,8 @@ step `Δt`, gravitational acceleration `g`, and free surface at time-step `n` `
 """
 function PCGImplicitFreeSurfaceSolver(grid::AbstractGrid, settings, gravitational_acceleration=nothing)
     # Initialize vertically integrated lateral face areas
-    ∫ᶻ_Axᶠᶜᶜ = Field{Face, Center, Nothing}(with_halo((3, 3, 1), grid))
-    ∫ᶻ_Ayᶜᶠᶜ = Field{Center, Face, Nothing}(with_halo((3, 3, 1), grid))
+    ∫ᶻ_Axᶠᶜᶜ = Field{Face, Center, Nothing}(grid)
+    ∫ᶻ_Ayᶜᶠᶜ = Field{Center, Face, Nothing}(grid)
```

Comment:
?

Reply:
a try to make it work before finding the real issue